test/externalTests/README.md

```diff
@@ -0,0 +1,66 @@
+## Solidity external tests
+This directory contains scripts for compiling some of the popular open-source projects using the
+current version of the compiler and running their test suites.
+
+Since projects often don't use the latest compiler, we keep a fork of each of these projects
+at https://github.com/solidity-external-tests/. If changes are needed to make a project work with the
+latest version of the compiler, they're maintained as a branch on top if the upstream master branch.
+This is especially important for testing our `breaking` branch because we can not realistically expect
+external projects to be instantly compatible with a compiler version that has not been released yet.
+It also isolates us from upstream changes to some degree - their changes won't affect our test suite
+until we explicitly pull them.
+
+### Recommended workflow
+
+#### Adding a new external project
+1. Create a fork of the upstream repository in https://github.com/solidity-external-tests/. If the
+    project consists of several repositories, fork them all.
+2. Remove all the branches except for main one (`master`, `develop`, `main`, etc). This branch is
+    going to be always kept up to date with the upstream repository and should not contain any extra
+    commits.
+3. Create two new versions of the main branch, representing versions of the compiler currently in
+    `develop` and `breaking`. E.g. if the latest Solidity version is 0.7.5 and the main branch of the
+    external project is called `master`, create `master_070` and `master_080`. This is where we will
+    be adding our own commits. The one corresponding to the newer Solidity version should always be
+    on top on the older one. E.g. if a change is needed to keep the project
```

Comment:
```suggestion
    on top of the older one. I.e. if a change is needed to keep the project
```
Maybe I'd also already say ``should always be *rebased* on top of the older one.``

By the way it's kind of sad that this situation ever occurs...

Reply:
I'll fix the `on` and `rebased` parts.

Regarding the `i.e.` vs `e.g.`, I actually meant the latter because it's an example using specific branch names.

> By the way it's kind of sad that this situation ever occurs...

I don't think we can expect anything else. Upgrades are extra work and they happen more often now. Some projects will always lag behind. It's good that at least some projects are moving away from 0.5.x now.